content/source/docs/enterprise/users-teams-organizations/users.html.md

```diff
@@ -4,47 +4,85 @@ page_title: "Users - Terraform Enterprise"
-they can be added to an organization.
+Users are the individual members of an [organization][organizations].
+
+## Creating an Account
+
+Users must create an account in Terraform Enterprise (TFE) before
+they can be added to an organization. Creating an account requires a username, an email address, and a password.
+
+The sign-up page is not linked from TFE's sign-in page, so you should provide the sign-up URL to any colleagues you want to invite to TFE:
+
+- For the SaaS version of TFE, create a new account at [https://app.terraform.io/account/new](https://app.terraform.io/account/new).
+- For private installs of TFE, go to `https://<TFE HOSTNAME>/account/new`.
+
+New users do not belong to any organizations.
+
+After you create a new user account, TFE immediately takes you to a page where you can create a new organization.
+
+![screenshot: the new organization page](./images/org-new.png)
+
+- If you are the first TFE user in your organization, use this page to create your TFE organization. See [Organizations][] for more information.
+- If you intend to join an existing organization, do not create a new one; instead, send your username to one of your TFE organization's owners and ask them to add you to a team. Once they have brought you into the organization, you can reload the page to begin using TFE.
 
 ## Team and Organization Membership
 
-Once a user has an account, an organization owner can add them to the
-organization by adding them to any of the organization's teams.
+To add a user to an organization, a member of that organization's owners team must add them to one or more teams. See [Teams][] for more information.
 
-To add a user to a team, navigate to the "Teams" section in your organization
-settings. Create a team if it does not already exist, or add the user to an
-existing team.
+Adding a user to a team requires only their username.
 
 ## User Settings
 
+-> **API:** See the [Account API](../api/account.html).
+
+TFE users can manage many of their own account details, including email address, password, API tokens, and two-factor authentication.
+
 To reach your user settings page, click the user icon in the upper right corner and choose "User Settings" from the menu.
 
-Once on this page, you can change your password and generate or revoke API tokens.
+Once on this page, can use the navigation on the left to choose which settings to manage.
 
 ![The user settings menu item, in the upper right menu.](./images/user-settings.png)
 
+### Profile
+
+TFE user profiles are very small, consisting only of a username and an email address. You can change either of these from the "Profile" page of the user settings.
+
+~> **Important:** Changing your username can cause important operations to fail. This is because it is used in URL paths to various resources. If external systems make requests to these resources, you'll need to update them prior to making a change.
```

Comment:
I'm not sure this is true anymore, actually. We should get this confirmed.

Reply:
@hashicorp/tfe-rails-reviewers Would one of you be able to confirm whether changing one's username in TFE still incurs the risk of breaking other systems? As I understand it, that should no longer be the case in V2.

Reply:
You can't edit your username if SAML is enabled, so that part is okay. :)
But yeah, I think you're probably right to stick to the warning here for now.